test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -1400,6 +1400,24 @@ TEST_F(AtenXlaTensorTest, TestDropoutInPlace) {
   ExpectCounterChanged("xla::bernoulli", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestNativeDropout) {
+  torch::Tensor a = torch::rand({17, 21}, torch::TensorOptions(torch::kFloat));
+  ForEachDevice([&](const torch::Device& device) {
+    torch::Tensor xla_a = CopyToDevice(a, device);
+    auto [xla_b_val, xla_b_mask] = torch::native_dropout(xla_a, 0.1, /*train=*/true);
```

Comment:
`p` is the probability that the elements will be zeroed out. So I think we should check a range around `p` instead of `1 - p`

Reply:
Cool! updated with 
```
EXPECT_GT(prob, 0.06);
EXPECT_LT(prob, 0.14);
```
And now the condition passed.